soco/core.py

```diff
@@ -828,6 +829,40 @@ def night_mode(self, night_mode):
             ('DesiredValue', int(night_mode))
         ])
 
+    @property
+    def surround_mode(self):
+        """bool: The speaker's surround mode.
+
+        True if on, False if off, None if not supported.
+        """
+        if not self.is_soundbar:
+            return None
+
+        response = self.renderingControl.GetEQ([
+            ('InstanceID', 0),
+            ('EQType', 'SurroundMode')
```

Comment:
This eqtype exists on my beam, but the phone app's "surround speaker" setting controls `SurroundEnable` instead of this (on S2 setup). This value controls the music level setting as shown here https://support.sonos.com/s/article/4804?language=en_US .